### DISCUSSION TITLE
General Questions

### DISCUSSION THREAD

**Discussion by salikadave** [salikadave/react-workshop-start-here#36] [General]
Hello!

We created this space so that you can ask us for any kind of advice / guidance on anything. Yes, it can be anything from DSA to interviews to certifications, anything that you are confused about!

However, before posting your questions, please check if your question is similar to any of the questions that have already been asked.

**Comment by yashmashru** [comment]
Hi salika, actually after you teach JS React , i just thought i want to do the best... I want to learn full stack development, but i need help, i am fully confused where to start, so you are much experienced so can you please just tell me what should i do , where to start , and ik I'm late but it's just a best time to start

**Reply by salikadave** [reply]
@yashmashru We're glad you reached out!

In @ShwetaNadar and I's opinion, we'd suggest the following:

1. Learn **JS** as much as we taught you, links are available on the repo. Every concept, practice the example given on MDN and write one example of your own.

2. Once you're familiar with some level of JS, build websites using **HTML, CSS, JS**.  This is essentially called as building websites with Vanilla JS. You can try cloning existing websites or build your own projects. [Reach out again if you can't find project ideas]. Make short small projects that make you understand **DOM manipulation**.
Here's a [link from The Odin Project](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/dom-manipulation) that will get you started with DOM Manipulation.

3. Once you're comfortable with this, learn **React**, first revise the concepts taught then do the [tutorial from the React Docs](https://reactjs.org/tutorial/tutorial.html). Then move on to making a project alongside this crash course from [Maximilian @ Academind](https://www.youtube.com/watch?v=Dorf8i6lCuk).

4. Once you're familiar with some level of JS, start learning Node.JS and exploring databases. If you're new to databases, try out MongoDB. *Reach out if you need help here!*

This path will make you work towards being a MERN Stack Developer. But if you prefer Angular over React, go for the MEAN Track
**M - MongoDB
E - Express
R/A - React / Angular
N - Node.js**

As long as your doing, it's never late :)

@everyone, reach out here if you're still confused.
